### ISSUE TITLE
VVV under Hyper-V

### ISSUE THREAD

**Issue by benlumia007** [Varying-Vagrant-Vagrants/VVV#1149]
I see that the hyper-v is supported, how would I get this working, do i need to change something for the provider and it should work correct.

**Comment by LoreleiAurora** [comment]
if hyper-v is setup on your machine just run `vagrant up --provider=hyperv`

**Comment by benlumia007** [comment]
hello @LoreleiAurora 

i did that, but it says that no usable default provider could be found in your system, but i do have hyper-v installed

**Comment by jeremyfelt** [comment]
@benlumia007 Are you running Windows 8.1 and have you enabled Hyper-V per the instructions here https://www.vagrantup.com/docs/hyperv/?

**Comment by benlumia007** [comment]
@jeremyfelt 
nope, i hope no one is using windows 8.1, im running Windows 10 Pro which has Hyper-V

**Comment by jeremyfelt** [comment]
@benlumia007 It seems like this may be an issue with Vagrant rather than VVV specifically. Are you running `vagrant` as an admin? From https://github.com/mitchellh/vagrant/issues/8346 and https://github.com/mitchellh/vagrant/issues/7853 I gather that this is necessary.

**Comment by benlumia007** [comment]
couldn't get it to work still. its ok, you can closed this ticket for now. I just thought it would be easy just to do vagrant up --provider=hyperv  but if there's some other configuration that needs to be set, then I will just use virtualbox for now.

**Comment by cfoellmann** [comment]
I tried this, too.
But I am using virtualbox and didn't really need this. I suspect that when you have (had?) virtualbox installed it causes issues for hyper-v.

**Comment by benlumia007** [comment]
@cfoellmann 
the only way to use Hype-V if you have Windows 8 Pro, Windows 8.1 Pro or Windows 10 Pro. I only have one Windows 10 Pro with Surface Pro 4 and I try using VVV for Hyper-V but i just hoped that it was easier enough to do the vagrant up but then is not, I think you have to do some other configurations for it to work.  

I think VVV is mainly focuses on VirtualBox more than other providers.

**Comment by cfoellmann** [comment]
@benlumia007 since most devs don't need the highest performance so virtualbox THE free option is the best choice to focus on.
I meant virtualbox and hyper-v on one machine = problems (at least for me)

**Comment by jeremyfelt** [comment]
> I think VVV is mainly focuses on VirtualBox more than other providers.

@benlumia007 Absolutely. We've added support for other providers, but the community using VirtualBox and providing feedback/changes is much larger (so far).

**Comment by benlumia007** [comment]
@jeremyfelt 
I definitely agree with that and plus most of the other provider aren't free or it only available in a pro version like Windows 8, 8.1 or 10 Pro. but then again, if VirtualBox is widely use and its absolutely free and use whatever or however you want. I moved my pro version to virtualbox since it is supported by default. 

But I realized when using VVV 2.0,  i have some broken links/urls that cannot find certain plugins like jetpack. since now there is a public_html folder now.
